presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -703,6 +746,36 @@ private static void createOrcAcidFile(File file)
         return splits.build();
     }
 
+    private static BackgroundHiveSplitLoader backgroundHiveSplitLoader(
```

Comment:
Could you extend `io.prestosql.plugin.hive.TestBackgroundHiveSplitLoader#backgroundHiveSplitLoader(io.prestosql.plugin.hive.HdfsEnvironment, io.prestosql.spi.predicate.TupleDomain<io.prestosql.plugin.hive.HiveColumnHandle>, java.util.Optional<io.prestosql.plugin.hive.util.HiveBucketing.HiveBucketFilter>, io.prestosql.plugin.hive.metastore.Table, java.util.Optional<io.prestosql.plugin.hive.HiveBucketHandle>, java.util.Optional<org.apache.hadoop.hive.common.ValidWriteIdList>)` version of `backgroundHiveSplitLoader` with dynamic filter parameters and add an override that does provide default values for these?

Reply:
done